recipe/meta.yaml

```
@@ -1,33 +1,32 @@
 {% set name = "pyerfa" %}
-{% set version = "2.0.0" %}
+{% set version = "2.0.1.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: f904231e1a570f94440e06140799590895107f942847b52a753ce81c9609162d
+  sha256: acb8a6713232ea35c04bc6e40ac4e461dfcc817d395ef2a3c8051c1a33249dd3
 
 build:
   number: 0
-  skip: true  # [py<37]
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
+    - python
     - {{ compiler('c') }}
   host:
     - python
     - pip
     - wheel
-    - setuptools >=42
-    - setuptools_scm >=3.4
+    - setuptools
+    - setuptools_scm >=6.2
     - jinja2 >=2.10.3
-    - numpy
+    - numpy >=1.19
```

```suggestion
    - numpy {{ numpy }}
```
We want to use the cbc version. When using >=1.19 the latest will be solved (1.26). Then pin_compatible in run will result in >=1.26,<2. Using the cbc will yield a higher compatibility range.

---

I'm having an issue on Python 3.9 when I change numpy via your suggestion
`ct.h:198:9: error: unknown type name 'vectorcallfunc'
          vectorcallfunc vectorcall;
          ^
  erfa/ufunc.c:33:1: warning: typedef requires a name [-Wmissing-declarations]
  typedef struct _typeobject {`

---

Reason was numpy 1.21 using vectorcallfunc from the python API, but pyerfa setting use of python limited API only. vectorcallfunc was added to the python limited api from python 3.12 only. (upstream made changes in that area for numpy2 compatiblity, this error might stem from those changes). Workaround: building with numpy 1.22.